hooks/update-status

```diff
@@ -1,22 +1,6 @@
 #!/usr/bin/env python3
 """Nrpe update-status hook"""
 
-import os
-import subprocess
-from charmhelpers.core.hookenv import status_set
-from services import get_revision
+import services
 
-SERVICE = "nagios-nrpe-server"
-
-
-def update_status():
-    """Update Nrpe Juju status."""
-    retcode = subprocess.call(["systemctl", "is-active", "--quiet", SERVICE])
-    if retcode == 0:
-        status_set("active", "Ready{}".format(get_revision()))
-    else:
-        status_set("blocked", "{} service inactive.".format(SERVICE))
-
-
-if __name__ == '__main__':
-    update_status()
+services.update_status()
```

Comment:
I used this pattern to be consistent with the other hook files.

Reply:
Isn't a good practice to keep the `if __name__ == '__main__':` ?

Not sure if this is imported somewhere else, but it can possibly trigger

Reply:
Probably, but yeah this file won't be imported by anything else (and with the filename, I don't think it can be imported). I updated it to be consistent with how the other hook files are written.